### ISSUE TITLE
Related Token Issue

### ISSUE THREAD

**Issue by JineteDV** [Cockatrice/Magic-Token#46]
Selecting all tokens option for cards like Dowsing Dagger, Huntmaster of the Fells, and treasure map doesn’t also make a token of the backside and attach the card to the token. Selecting all tokens for Arlinn Kord or Gideon, Ally of Zendikar doesn’t even make their emblems.
~~Doubling clicking on treasure map puts it in the land row instead of the noncreature and nonland row.~~

**Comment by tooomm** [comment]
> Doubling clicking on treasure map puts it in the land row instead of the noncreature and nonland row.

I opened https://github.com/Cockatrice/Cockatrice/issues/2904, that's not a token issue.

**Comment by Psithief** [comment]
The goal was to prevent the shortcut from creating objects that would not be possible under the game rules for a single action.

Emblems have been intentionally excluded from the All Tokens shortcut because they typically cannot be produced in the same action as creating tokens from a Planeswalker card. Likewise, attaching is also excluded. Then there's some logic I don't quite remember about defaulting to use relations marked as excluded if there are no normal relations.

tl;dr: I put a lot of thought into this and it's working exactly as designed.
